packages/experience/src/hooks/use-webauthn-operation.ts

```diff
@@ -63,19 +63,26 @@ const useWebAuthnOperation = () => {
         }
       );
 
-      if (response) {
-        /**
-         * Assert type manually to get the correct type
-         */
-        void sendMfaPayload(
-          isAuthenticationResponseJSON(response)
-            ? {
-                flow: UserMfaFlow.MfaVerification,
-                payload: { ...response, type: MfaFactor.WebAuthn },
-              }
-            : { flow: UserMfaFlow.MfaBinding, payload: { ...response, type: MfaFactor.WebAuthn } }
-        );
+      if (!response) {
+        return;
       }
+
+      /**
+       * Assert type manually to get the correct type
+       */
+      void sendMfaPayload(
+        isAuthenticationResponseJSON(response)
+          ? {
+              flow: UserMfaFlow.MfaVerification,
+              payload: { ...response, type: MfaFactor.WebAuthn },
+              verificationId,
+            }
+          : {
+              flow: UserMfaFlow.MfaBinding,
+              payload: { ...response, type: MfaFactor.WebAuthn },
+              verificationId,
+            }
```

Comment:
```ts
{
  flow: isAuthenticationResponseJSON(response)
    ? UserMfaFlow.MfaVerification : UserMfaFlow.MfaBinding,
  payload: { ...response, type: MfaFactor.WebAuthn },
  verificationId,
}
```

Reply:
Had a similar concern. Looks like the input type is a union, this shorthand code can not infer types properly.

Reply:
ok then